### DISCUSSION TITLE
How can I use vue-devtools for developing an extension?

### DISCUSSION THREAD

**Discussion by barakplasma** [directus/directus#14359] [Archive]
I'm developing a custom layout extension in Directus, and would like to use the vue-devtools while testing it. However, after building the extension and copying it into Directus, the extension is running Vue in production mode. I think it's related to how the @directus/extensions-sdk build command https://github.com/directus/extensions-sdk/blob/d4a51726300c07d302a918923c6c5bb78ecef6da/src/cli/commands/build.ts#L150 configures Vue. Is there some other way to build the extension in development mode?

**Comment by br41nslug** [comment]
To start the app in development mode currently the only way is to run it locally using these instruction https://docs.directus.io/contributing/running-locally/
